doc/classes/TabBar.xml

```diff
@@ -140,6 +140,18 @@
 				Removes the tab at index [param tab_idx].
 			</description>
 		</method>
+		<method name="select_next_available">
+			<return type="bool" />
+			<description>
+				Selects first available tab on the right from the currently selected. Returns [code]true[/code] if tab selection changed.
```

Comment:
I think UI mirroring will contradict this, as with a RTL layout the "next" tab will be to the left. So this may need to be rephrased to avoid this ambiguity (same in TabContainer). CC @bruvzg

Reply:
Indeed, in RTL layout, "next" is "left".

Reply:
```suggestion
				Selects the first available tab on the right from the currently selected. Returns [code]true[/code] if tab selection changed.
```
Same below

Reply:
> Oh, in that case we have a bigger issue, as pressing left on keyboard or controller will go in opposite direction in RTL. I wasn't aware tabs are reordered in this way.

The fix should be pretty simple, you would need to check `is_layout_rtl()` and use the opposite direction for the target tab. It would be fine to have a bit of code repetition between the next and the previous tab method.

Reply:
I'm having a little problem with rephrasing that sentence to be grammatically correct. How does
"Selects the first available tab with greater/lower index than the currently selected." sounds like? In case of lower index it sounds like the function will always select first tab ever so I'm not sure about this one.